### PR TITLE
Add otherwise route, minor changes

### PR DESCRIPTION
**Pull request by yx296**
Add some clarifying comments for routes.js, Just testing a small PR.

### CODE REVIEW

File: client/vivace/list/routes.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 let paths = ['/', '/list/:listType'];
 
 module.exports = paths.map((path) => {
-  path = {path};
+  path = {path}; // es6 object initialization compiles to path = { path: path };
```
Comment (jackofseattle) [inline client/vivace/list/routes.js]: ```module.exports = paths.map( ({path}) => {```

you can destructure inside of the argument block - neat, huh?

File: client/vivace/main-nav/directive.js [hunk 2]
```diff
@@ -6,16 +6,16 @@ module.exports = ($window) => {
     templateUrl: template,
     controller: 'mainNavController',
     controllerAs: 'mainNav',
-    link: ($scope, $el) => {
-      let getNavHeight =  ($scope, $el) => {
-        $scope.mainNav.height = $el.prop('offsetHeight').toString() + 'px';
+    link: (scope, el) => {
```
Comment (jackofseattle) [inline client/vivace/main-nav/directive.js]: prefer to leave $ in place for angular injected args.

File: client/vivace/main-nav/directive.js [hunk 3]
```diff
@@ -6,16 +6,16 @@ module.exports = ($window) => {
...
+        scope.mainNav.height = el.prop('offsetHeight').toString() + 'px';
       };
 
-      getNavHeight($scope, $el);
-      angular.element($window).bind('resize', () => {
-        getNavHeight();
-        $scope.$digest();
-      });
+      getNavHeight(scope, el);
+      // angular.element($window).bind('resize', () => {
```
Comment (jackofseattle) [inline client/vivace/main-nav/directive.js]: Please don't comment out code unless you have a good reason.  Either delete it or leave it and refactor.

**Comment by jackofseattle** [comment]
A few small notes; good first PR.

File: client/vivace/main-nav/directive.js (hunk 3 above)
Comment (yx296) [inline client/vivace/main-nav/directive.js]: Yeah, those 4 lines were throwing up an error in console:
Uncaught TypeError: Cannot read property 'mainNav' of undefined 
that was being triggered by any resizing of the window(just opening chrome dev tools triggers it)
but I wasn't sure what it was trying to do so I left it in

Thanks!

Comment (jshor) [inline client/vivace/main-nav/directive.js]: I think it's weird that we did $scope.$digest()` in a `$window` resize event

**Comment by jshor** [comment]
LGTM but I recommend removing the commented-out code

File: client/vivace/main-nav/directive.js (hunk 3 above)
Comment (realalexhomer) [inline client/vivace/main-nav/directive.js]: This is so that when you resize the nav and the height changes, the nav's position changes as well.

Comment (realalexhomer) [inline client/vivace/main-nav/directive.js]: If you want I can look at what is causing the error, it shouldn't be hard to fix.

Comment (yx296) [inline client/vivace/main-nav/directive.js]: Ok sure, I'll wait for that before updating my PR.
Thanks



On Thu, Jan 28, 2016 at 9:41 AM, Alex Homer <notifications@github.com>
wrote:

> In client/vivace/main-nav/directive.js
> <https://github.com/inrhythm/vivace/pull/143#discussion_r51157160>:
>
> >        };
> >
> > -      getNavHeight($scope, $el);
> > -      angular.element($window).bind('resize', () => {
> > -        getNavHeight();
> > -        $scope.$digest();
> > -      });
> > +      getNavHeight(scope, el);
> > +      // angular.element($window).bind('resize', () => {
>
> If you want I can look at what is causing the error, it shouldn't be hard
> to fix.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inrhythm/vivace/pull/143/files#r51157160>.
>

-- 
 InRhythm™ <http://www.inrhythm.com>  <http://www.inrhythm.com/> |  Services 
<http://inrhythm.com/resources/Brochure_Inrhythm.pdf> | Testimonials 
<http://inrhythm.com/testimonials.html>

494 8th Ave. Suite 1700 New York, NY 10001

*Accelerating product delivery *Follow Us <https://twitter.com/GetInRhythm>

**Comment by yx296** [comment]
Removed commented code, put $ back into scope and el parameters for the mainNav function

**Pull request merged**